packages/main/test/sap/ui/webcomponents/main/samples/Input.sample.html

```diff
@@ -140,6 +151,42 @@ <h3>Input with Value State</h3>
 		</xmp></pre>
 	</section>
 
+	<section>
+		<h3>Input as Search Field</h3>
+		<div class="snippet">
+			<div class="flex-column samples-margin">
+				<ui5-input id="searchInput" placeholder="Enter search criteria ...">
+					<ui5-icon id="searchIcon" data-ui5-slot="icon" src="sap-icon://search" class="inputIcon"></ui5-icon>
+				</ui5-input>
+			</div>
+		</div>
+		<pre class="prettyprint lang-html"><xmp>
+<style>
```

Comment:
Duplicate

Reply:
> Style tag and its content is duplicate

Style tag is for the docs readers, information how to style the icon appearance